care/emr/api/viewsets/scheduling/token.py

```
@@ -107,6 +108,12 @@ def perform_update(self, instance):
                 if obj.sub_queue.current_token == obj:
                     obj.sub_queue.current_token = None
                     obj.sub_queue.save(update_fields=["current_token"])
+            elif (
+                instance.sub_queue and instance.sub_queue.resource != obj.queue.resource
+            ):
+                raise ValidationError(
+                    "Sub Queue and Queue are not in the same resource"
+                )
```

Ruff (0.14.5)

Avoid specifying long messages outside the exception class

(TRY003)